server/util.go

```diff
@@ -44,26 +44,26 @@ func AddCommands(
 	appExport types.AppExporter,
 	addStartFlags types.ModuleInitFlags,
 ) {
-	tendermintCmd := &cobra.Command{
-		Use:   "tendermint",
-		Short: "Tendermint subcommands",
+	cometCmd := &cobra.Command{
+		Use:   "cometbft",
```

Comment:
We are changing the command family `tendermint` to `cometbft`. That requires appropriate adjustments in the v-kit repository. Not something for now.

Reply:
I reverted this change in c4d2fcda8ce7709d867aeead956752d518211eb9. We can merge everything else if we want for smaller steps and address this one later.